webapp/portlet/src/main/webapp/vue-apps/general-settings/components/branding/form/BorderRadiusSelector.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="d-flex mt-8">
+    <v-btn 
+      text
+      small
+      @click="setBorderRadius(0)">
+      {{ $t('generalSettings.widgetAndAppStyle.square') }}
+    </v-btn>
+    <v-slider
+      v-model="defaultBorderRadius"
+      :thumb-size="24"
+      tick-size="2"
+      thumb-label="always"
+      ticks
+      min="0"
+      max="20"
+      step="4"
+      color="primary"
+      @change="$emit('input', defaultBorderRadius)" />
```

Comment:
No need when using `v-model`
```suggestion
/>
```

Reply:
done